examples/tutorials/core_api_pytorch_mnist/model_def_distributed.py

```diff
@@ -234,7 +243,7 @@ def main(core_context):
     optimizer = optim.Adadelta(model.parameters(), lr=hparams["learning_rate"])
     scheduler = StepLR(optimizer, step_size=1, gamma=args.gamma)
 
-    starting_epoch = 0
```

Comment:
this was always a bug, since you have a zero-indexed `epoch` but you calculate `steps_completed = epoch * len(training_loader)`, which will be zero after you train one epoch.

Again, my recommendation about being explicit naming `epoch_idx` and `epochs_completed` helps avoid these off-by-one errors.

Side note, there's no reason for this `starting_epoch` variable, prolly get rid of it.